transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollStreamChannel.java

```diff
@@ -88,6 +75,7 @@
 
     private WritableByteChannel byteChannel;
 
+    @Deprecated
```

Comment:
Usages of this method outside Netty will have no replacement though ... what should the comments indicate for users?